rg_instructor_analytics/static/rg_instructor_analytics/js/Base.js

```diff
@@ -1,3 +1,11 @@
+(function() {
+    /*
+    Gets favicon from body and up it to head.
```

Comment:
Typo:  s/Gets favicon from body and up it to head./Get favicon from the <body> and add it to the <head>./

Reply:
Done